### DISCUSSION TITLE
npm issues  WARN deprecated when installing serverless version 3.7.4

### DISCUSSION THREAD

**Discussion by johnzoet** [serverless/serverless#10844] [Q&A]
When I install serverless 3.7.4 I get several warnings.
![warning-deprecated-serverless-3 7 4](https://user-images.githubusercontent.com/9982429/157968753-98d1f933-7ca1-4c47-97a8-c26d1427b060.jpg)

Upgrade instructions at:
https://www.serverless.com/framework/docs/guides/upgrading-v3#:~:text=The%20simplest%20way%20to%20upgrade%20to%20v3%20is,any%20deprecations%20can%20be%20immediately%20upgraded%20to%20v3.

I tried to upgrade from version 2 to version 3 but got these warning messages.
Then I uninstalled serverless and re-installed serverless but still get these warning messages.
Is there an issue on my PC?

**Comment by Knufle** [comment]
No, I don't think so, these are only warnings telling you that some of the dependencies used by this version of serverless could be updated/upgraded or even removed, it's not related to any project of yours

**Reply by johnzoet** [reply]
I tried to upgrade the dependencies but that did not work either.
However the solution that is working for me is to:
- uninstall serverless by npm: npm uninstall serverless
- install serverless via chocolatey: choco install serverless

Now serverless is working great.
